### PR TITLE
MWPW-126640: consume standalone OST

### PR DESCRIPTION
**Pull request by 3ch023**
This PR add ability to configure checkoutType and worfklowStep on OST CTA link.
It will also consume OST from web stage instead of copying js to Milo sources.

Resolves: [MWPW-126640](https://jira.corp.adobe.com/browse/MWPW-126640)

**Test URLs:**
remember to set 'AOS_ACCESS_TOKEN'.

- Before: [OST](https://ost--milo--adobecom.hlx.page/tools/ost?offerId=257E1D82082387D152029F93C1030624&type=checkoutUrl&text=buy-now&checkoutType=UCv3&workflowStep=email)
- After: [OST](https://ost--milo--3ch023.hlx.page/tools/ost?offerId=257E1D82082387D152029F93C1030624&type=checkoutUrl&text=buy-now&checkoutType=UCv3&workflowStep=email)

image

### CODE REVIEW

File: libs/blocks/ost/ost.js [hunk 1]
```diff
@@ -5,19 +5,19 @@ const base = miloLibs || codeRoot;
 
 export default async function init() {
   if (!window.ost) {
-    await loadScript(`${base}/deps/offer-selector-tool.js`);
+    await loadScript(`https://www.stage.adobe.com/special/tacocat/ost/offer-selector-tool.js`);
```
[the remark this answers is not in the document]
Comment (3ch023) [reply libs/blocks/ost/ost.js:8]: for now, we decided to always pull OST from the stage only, yes.
main reason was that it's hidden under a vpn and we are not yet sure about exposing ost code to the public.

**Review by yesil — COMMENTED** [review]
ost styles should be loaded from the same domain
offer-selector-tool.css
OST looks broken to me.

**Comment by 3ch023** [comment]
> ost styles should be loaded from the same domain offer-selector-tool.css OST looks broken to me.

@yesil i fixed it with @vhargrave  you can check again [here](https://ost--milo--3ch023.hlx.page/tools/ost?offerId=257E1D82082387D152029F93C1030624&type=checkoutUrl&text=buy-now&checkoutType=UCv1&workflowStep=checkout_email)

So I guess [https://ost--milo--3ch023.hlx.page/libs/blocks/ost/ost.css](https://ost--milo--3ch023.hlx.page/libs/blocks/ost/ost.css) got requested too early and franklin cached the 404 error.
we tried `curl -X PURGE "{LIVE_URL}"` but it didn't do any effect.
so i just committed a change on ost.css and it started loading properly.

File: libs/blocks/ost/ost.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (honstar) [reply libs/blocks/ost/ost.js:8]: We definitely need to review this to avoid making corp network or VPN a requirement to use OST.

**Review by honstar — APPROVED** [review]
LGTM

**Review by Axelcureno — APPROVED** [review]

**Comment by yesil** [comment]
@3ch023 this is a PR to `ost` branch. Can you sync up with @Axelcureno when you can get this merged into the main branch?

**Comment by 3ch023** [comment]
> @3ch023 this is a PR to `ost` branch. Can you sync up with @Axelcureno when you can get this merged into the main branch?

yeah, it's just a temp PR to not lose our changes on ost.
waiting for an IMS to be ready before we can PR main branch.

File: libs/blocks/ost/ost.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (npeltier) [reply libs/blocks/ost/ost.js:8]: i think this ideally should be a requirement

**Pull request merged**